### DISCUSSION TITLE
GitHub actions - poe command not found

### DISCUSSION THREAD

**Discussion by songololo** [nat-n/poethepoet#249] [Q&A] [answered]
I'm trying to use `poe` from Github Actions but in spite of installing the package I get a `poe: command not found` error message.

Any advice for setup and use in the below Actions context?

```bash
jobs:
  check:
    runs-on: ubuntu-latest
    strategy:
      matrix:
        python-version: ["3.10", "3.11", "3.12"]
    steps:
      - name: Checkout code
        uses: actions/checkout@v4
      - name: Install uv
        uses: astral-sh/setup-uv@v3
      - name: Set up Python ${{ matrix.python-version }}
        run: uv python install ${{ matrix.python-version }}
      - name: Verify project
        run: |
          uv sync --dev
          poe verify_project
```

**Comment by songololo** [comment, marked as answer]
For others running into this, the answer is to run `poe` using the `uv run` command, so `uv run poe` should do the trick.

**Comment by nat-n** [comment]
I'm glad you found a solution :)

I usually use pipx for setup like so: https://github.com/nat-n/homebrew-poethepoet/blob/main/.github/workflows/update-formula.yaml#L19 
Or exceptionally in this project I run poe via poetry: https://github.com/nat-n/poethepoet/blob/main/.github/workflows/publish-docs.yml#L29
